### ISSUE TITLE
Audio controls are broken on "develop" branch, current deployment works

### ISSUE THREAD

**Issue by tabascoeye** [thecodingmachine/workadventure#1440]
See on my tutorial-map. Walk to the big yellow bell.

Scenario 1:
https://play.workadventu.re/_/global/raw.githubusercontent.com/workadventure-xce/workadventure-xce-map-starter-kit/main/map.json
* controls work (mute/volume/checkbox)
* only "mute" status is "saved" when you exit the playAudio zone then re-enter

Scenario 2:
https://play-twemojiemotemenusvelte.test.workadventu.re/_/global/raw.githubusercontent.com/workadventure-xce/workadventure-xce-map-starter-kit/main/map.json
* no controls work

Also on a map where the start zone lies **inside** a playAudio zone, the Audio only plays when the whole login-screen-game is gone through (e.g. with a private browser window), but on each reload without all the name/character scenes, audio will not start playing

**Comment by moufmouf** [comment]
Indeed! Thanks a lot, we'll take a look at it!

**Comment by tabascoeye** [comment]
maybe add that to the long list of tests ;)

**Comment by moufmouf** [comment]
Good idea. Test added in #1445

**Comment by Lurkars** [comment]
Please reopen, I can still confirm that audio is no playing, when you start on a tile with Audio. Controls appear and are usable, but Audio is not playing.

Another issue I noticed, different issue on exit: when you exit a room where Audio is also on exit tile, the audio continues playing in the next map from exist, even trough there is no Audio tile.

**Comment by tabascoeye** [comment]
@moufmouf should we open a new ticket to address the remaining issues?
(doesn't play when start tile is in audiozone && doesn't stop playing when exit tile is reached)

**Comment by moufmouf** [comment]
Ok sorry, issue reopened for those 2 remaining bugs.

**Comment by moufmouf** [comment]
> I can still confirm that audio is no playing, when you start on a tile with Audio. Controls appear and are usable, but Audio is not playing.

There is nothing we can do about that alas.
Because no page can play a sound without prior user interaction. And since we removed the camera setup screen, you do arrive in the map without any interaction.

![image](https://user-images.githubusercontent.com/1290952/133439188-52b10334-f6cb-4523-a30b-95c2ba040abb.png)

**Comment by tabascoeye** [comment]
@moufmouf that needs a thorough explaination in the documentation about ´´´´playAudio´´´ because it will be confusing as hell to people who have that working in their maps already

**Comment by Lurkars** [comment]
> > I can still confirm that audio is no playing, when you start on a tile with Audio. Controls appear and are usable, but Audio is not playing.
> 
> There is nothing we can do about that alas.
> Because no page can play a sound without prior user interaction. And since we removed the camera setup screen, you do arrive in the map without any interaction.
> 
> ![image](https://user-images.githubusercontent.com/1290952/133439188-52b10334-f6cb-4523-a30b-95c2ba040abb.png)

Mhm is it easy to detect such state? Then maybe one can "disable" audio on start, and start playing with first movement of player.

**Comment by tabascoeye** [comment]
according to this: https://docs.agora.io/en/Voice/autoplay_policy_web?platform=Web
other people have the same challenge and solve it by either:
* starting the player as "muted" (then it is allowed to autoplay) and unmute it on interaction (after some eventcallback)
OR
* register a callback listener on some interaction event and only after that start playing audio

I guess it would be easy to do that in WA theoretically. Not sure about what would have to be done specifically to have the audio element be aware of e.g. Player movement
